File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Get height from user
    int height;
    do
    {
        height = get_int("Enter height: ");
    }
    while (height < 1 || height > 8);


    //Height of pyramid
    for (int i = 0; i < height; i++)
    {
        //Left spaces
        for (int j = height - 1; j > i; j--)
        {
            printf(" ");
        }
        //Left hashes
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        //Middle spaces
        if (i < height)
        {
            printf("  ");
        }

        //Right hashes
        for (int l = 0; l <= i; l++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int


# Get input from the user between 1-8
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break


# Height of pyramid
for i in range(height):

    # Left spaces
    for j in range(height - i - 1):
        print(" ", end="")

    # Left hashes and middle spaces
    for k in range(i + 1):
        print("#", end="")
    print("  ", end="")

    # Right hashes
    for l in range(i + 1):
        print("#", end="")
    print("")